src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/processedenvelopes/ProcessedEnvelope.java

```diff
@@ -6,8 +6,15 @@
 
     @JsonProperty
     public final String id;
+    @JsonProperty("processed_ccd_reference")
+    public final Long processedCcdReference;
+    @JsonProperty("processed_ccd_type")
+    public final EnvelopeCcdAction processedCcdType;
```

Comment:
shall we change names
id--> envelope_id
processed_ccd_reference -->  ccd_id
processed_ccd_type -> ccd_action

Reply:
processor firstly will need to start accepting `envelope_id` ;)

Reply:
will keep id too until remove from processor

Reply:
let's adjust processor first maybe, less work, fewer steps in the future

Reply:
we need decide the names as I will add the db columns in processor

Reply:
I think the new names you've suggested in the top comment are much clearer